config/main.py

```diff
@@ -858,45 +861,43 @@ def _stop_services():
     try:
         subprocess.check_call(['sudo', 'monit', 'status'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
         click.echo("Disabling container monitoring ...")
-        clicommon.run_command("sudo monit unmonitor container_checker")
+        clicommon.run_command(['sudo', 'monit', 'unmonitor', 'container_checker'])
     except subprocess.CalledProcessError as err:
         pass
 
     click.echo("Stopping SONiC target ...")
-    clicommon.run_command("sudo systemctl stop sonic.target --job-mode replace-irreversibly")
+    clicommon.run_command(['sudo', 'systemctl', 'stop', 'sonic.target', '--job-mode', 'replace-irreversibly'])
 
 
 def _get_sonic_services():
-    out, _ = clicommon.run_command("systemctl list-dependencies --plain sonic.target | sed '1d'", return_cmd=True)
+    cmd = "systemctl list-dependencies --plain sonic.target | sed '1d'"
+    out, _ = clicommon.run_command(cmd, return_cmd=True, shell=True)
```

Comment:
>cmd [](http://example.com/codeflow?start=35&length=3)

you may use one command `systemctl list-dependencies --plain sonic.target`.
And then parse the output in python instead of `sed`.